homeassistant/components/weather/openweathermap.py

```diff
@@ -47,6 +49,7 @@
     vol.Optional(CONF_LATITUDE): cv.latitude,
     vol.Optional(CONF_LONGITUDE): cv.longitude,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_FORECAST): cv.boolean
```

Comment:
I guess we can turn this always on, I'll remove the option.
I've also limited the forecast data requests once per 30 minutes instead of 10 minutes (for the current weather). Not sure if that's needed but I guess forecasts don't need to be updated as regularly